website/package.json

```diff
@@ -13,6 +13,7 @@
 	},
 	"dependencies": {
 		"async": "^2.5.0",
+		"clipboard": "^2.0.0",
```

Comment:
Since we're loading clipboard.js from CDN, we don't need to add it to the `package.json`.

Reply:
Ok. I think I am still a bit confused about how Docusaurus works on a `docusaurus-init` so I'll dig into that some more.